tutorial/10_key_features/004_distributed.py

```diff
@@ -55,4 +77,5 @@ def objective(trial):
 
```

Comment:
Could you explain that SQLite with multi-processes will make the errors of deadlock? Or, you can use the mysql in the example code.